src/System.DirectoryServices.AccountManagement/tests/System.DirectoryServices.AccountManagement.Tests.csproj

```diff
@@ -25,5 +25,8 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
This is created due to a bug in certain recent versions of VS and can be reverted

Reply:
I have the last VS 2017 community edition. Why does this bug stay in it?